components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/BrowserAction.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.webextension
+
+import android.graphics.drawable.Drawable
+
+/**
+ * Value type that represents the state of a browser action within a [WebExtension].
+ *
+ * @property title The title of the web extension to be visible in the user interface.
```

Comment:
This is the title of the browser action https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/setTitle not the extension.